src/main/java/org/folio/rest/impl/RtacResourceImpl.java

```diff
@@ -157,7 +163,8 @@ private JsonArray verifyAndExtractHoldings(Response response) {
     for (Holding holding : holdings.getHoldings()) {
       try {
         cfs.put(holding,
-            httpClient.request("/circulation/loans?limit=" + Integer.MAX_VALUE + "&query=%28itemId%3D%3D"
+            httpClient.request("/circulation/loans?limit="
+                + Integer.MAX_VALUE + "&query=%28itemId%3D%3D"
```

Comment:
I'm new to this codebase and I see these raw query strings used quite a bit already, but for the future would it make sense to use a library such as [URLEncoder](https://docs.oracle.com/javase/8/docs/api/java/net/URLEncoder.html) to make these strings easier to read and less prone to accidental typing errors?

Reply:
Agreed. I believe when this was written there was an issue around what was coming out of URLEncoder that made us preencode the query path. It might have been the %20 vs + for the space mapping. It should really use '+'. Maybe this is not an issue anymore or we were doing something wrong back then. It is worth re-investigating. I'll take a look at it for this PR.

Reply:
[Mod-circulation](https://github.com/folio-org/mod-circulation/blob/353da4b0686d93464090efcfabcc7b0cd4b32705/src/main/java/org/folio/circulation/support/CqlQuery.java#L88) uses URLEncoder, I still find it confusing and a tad frustrating it cannot just be applied to a whole URL rather than only the value of the query string parameters.